python/pyspark/sql/chunk_api.py

```diff
@@ -0,0 +1,126 @@
+#
```

Comment:
module name shouldn't better have `_` there especially for user-facing one. I would just rename it to `chunk` from `chunk_api`.